cmd/node/config/api.toml

```diff
@@ -65,7 +65,15 @@
 
         # /network/config will return metrics related to current configuration of the network (number of shards,
         # consensus group size and so on)
-        { Name = "/config", Open = true }
+        { Name = "/config", Open = true },
```

Comment:
`Open = false` by default? Considering your benchmarks, these endpoints can really slow down the node and we wouldn't want these endpoints called by mistake for a validator.

Reply:
added TODO to make them false as default after the initial testing round is over